examples/example-server-lib/sw_splash.h

```diff
@@ -28,6 +28,7 @@ class SwSplash
     SwSplash();
     ~SwSplash();
 
+    void operator()(bool splash_opt);
```

Comment:
I know other functions on this are done as `operator()`s, but this is probably better as a normal member function. E.g. `void enable(bool show_splash)`

Reply:
fixed in pr update